docs/cli/tkn-results_config_set.md

```diff
@@ -1,39 +1,38 @@
 ## tkn-results config set
 
-Configure Tekton Results CLI settings
+Set Tekton Results CLI configuration values
 
 ### Synopsis
 
-Configure settings for the Tekton Results CLI.
+Configure how the CLI connects to the Tekton Results API server.
 
-This command allows you to configure how the CLI interacts with the Tekton Results API server.
-It can automatically detect the API server in your cluster or allow manual configuration.
 
-The command will:
-1. Automatically detect the Tekton Results API server in your cluster
-2. Prompt for any missing configuration values
-3. Save the configuration for future use
 
-Automatic Detection:
-- Cluster context and namespace
-- API server endpoint
-- Service account token (if available)
+Usage Modes:
+1. Interactive: Prompts for values with defaults where available
+   tkn-results config set
```

Comment:
Is this supposed to be monospaced/code-format? The tab makes me think it is supposed to be, but it's rendering as paragraph text:
<img width="832" height="208" alt="image" src="https://github.com/user-attachments/assets/75f03812-f0fd-440b-bd3c-dda5a4af25d2" />